gui/services/migrations/0195_auto__add_field_nfs_nfs_srv_mountd_log__add_field_nfs_nfs_srv_statd_lo.py

```diff
@@ -11,7 +11,7 @@ def forwards(self, orm):
         # Adding field 'NFS.nfs_srv_mountd_log'
         db.add_column(u'services_nfs', 'nfs_srv_mountd_log',
                       self.gf('django.db.models.fields.BooleanField')(default=True),
-                      keep_default=False)
+                      keep_default=True)
 
```

Comment:
Ok William, I'll change the values as integers. 
This migration file reflected from /services/models.py, and I had set the values there also as 'default=True'.
Should I make this change there also or it should be only in migration file?